source/main/utils/HandleDiskSpace.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import { BrowserWindow, ipcMain } from 'electron';
+import checkDiskSpace from 'check-disk-space';
+import prettysize from 'prettysize';
+import { GET_DISK_SPACE_STATUS } from '../../common/ipc-api';
+import { environment } from '../environment';
+import { Logger } from './logging';
+import {
+  DISK_SPACE_REQUIRED,
+  DISK_SPACE_REQUIRED_MARGIN_PERCENTAGE,
+  DISK_SPACE_CHECK_LONG_INTERVAL,
+  DISK_SPACE_CHECK_SHORT_INTERVAL,
+  DISK_SPACE_RECOMMENDED_PERCENTAGE
+} from '../config';
+
+export default (
+  mainWindow: BrowserWindow,
+  onCheckDiskSpace?: Function,
+) => {
+  const path = environment.isWindows ? 'C:' : '/';
```

Comment:
Maybe there is a better way to let node.js handle the os differences?

Reply:
I couldn't find a way. I got this code from one the [packages](https://www.npmjs.com/package/diskusage) that check disk space.